### PR TITLE
Revert testIf changes

### PR DESCRIPTION
**Pull request by angrykoala**
# Description
This PR reverts the addition of `testIf` as it is not really working for the intended case

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Made some changes to the GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at https://neo4j-graphql-toolbox-stage-3145.surge.sh

Use for instance a local dbms hosted on [Neo4j Desktop](https://neo4j.com/download/) or a cloud-hosted [Aura Free](https://neo4j.com/cloud/aura-free/) dbms.

File: packages/graphql/tests/integration/directives/fulltext/fulltext-index.int.test.ts [hunk 1]
```diff
@@ -323,7 +348,7 @@ describe("@fulltext directive - indexes constraints", () => {
         ).rejects.toThrow(`@fulltext index '${indexName}' on Node '${type.name}' is missing field 'description'`);
     });
 
-    testIf(MULTIDB_SUPPORT)("should throw when index is missing fields (using field alias)", async () => {
+    test("should throw when index is missing fields (using field alias)", async () => {
```
Comment (darrellwarde) [inline packages/graphql/tests/integration/directives/fulltext/fulltext-index.int.test.ts:351]: Missing the `MULTIDB_SUPPORT ` conditional?

File: packages/graphql/tests/integration/directives/fulltext/fulltext-index.int.test.ts [hunk 2]
```diff
@@ -474,7 +511,7 @@ describe("@fulltext directive - indexes constraints", () => {
         );
     });
 
-    testIf(MULTIDB_SUPPORT)("should create index for ID field if it doesn't exist", async () => {
+    test("should create index for ID field if it doesn't exist", async () => {
```
Comment (darrellwarde) [inline packages/graphql/tests/integration/directives/fulltext/fulltext-index.int.test.ts:514]: Also missing the `MULTIDB_SUPPORT ` conditional?

**Review by darrellwarde — COMMENTED** [review]
Sorry you have to revert this one... 😢

**Pull request merged**